packages/components/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@codesandbox/components",
+  "version": "1.0.8",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/codesandbox/codesandbox-client",
+    "directory": "packages/components"
+  },
+  "files": [
+    "lib"
+  ],
+  "source": true,
+  "scripts": {
+    "build": "yarn build:lib",
+    "build:dev": "yarn build",
+    "build:lib": "yarn clean && yarn tsc && yarn cpx \"src/**/*.{css,svg,png,jpg,woff,woff2}\" lib",
+    "build:storybook": "build-storybook -c .storybook -o public",
+    "clean": "rimraf lib && yarn rimraf node_modules/@types/react && yarn rimraf node_modules/@types/react-native",
+    "lint": "eslint --ext .js,.ts,.tsx src",
+    "prepublish": "yarn build",
+    "start": "(yarn tsc --watch & yarn cpx \"src/**/*.{css,svg,png,jpg,woff,woff2}\" lib --watch)",
+    "start:storybook": "start-storybook",
+    "typecheck": "tsc --noEmit",
+    "chromatic": "CHROMATIC_APP_CODE=nffds42ndde ./node_modules/.bin/chromatic --build-script-name=build:storybook"
+  },
+  "dependencies": {
+    "@codesandbox/common": "^1.0.8",
+    "@codesandbox/template-icons": "^1.1.0",
+    "@styled-system/css": "^5.1.4",
+    "codesandbox-api": "0.0.23",
+    "color": "3.1.2",
+    "date-fns": "^2.8.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-router-dom": "^5.1.2",
+    "styled-components": "^4.4.1",
```

Comment:
oh my god, are you saying we can use a different styled-component version/library than the app? Is that a thing?

Reply:
I mean it gets compiled, I don't see why